packages/tldraw/src/lib/ui/components/DebugPanel.tsx

```diff
@@ -68,6 +82,39 @@ const CurrentState = track(function CurrentState() {
 	return <div className="tlui-debug-panel__current-state">{editor.getPath()}</div>
 })
 
+const times: number[] = []
+let isFPSCounterRunning = false
+let fps = 0
+function refreshFpsLoop() {
+	window.requestAnimationFrame(function () {
```

Comment:
we have a batched requestAnimationFrame called `throttledRaf` that we use for performing store changes. I think you'd get more accurate results by using that. With this code as-is I believe it's actually testing how many extra animation frames it can fit between the frames we actually paint on?

Reply:
actually forget my advice, I think it'll work as intended.